CMakeLists.txt

```diff
@@ -338,8 +350,8 @@ if (CGNS_ENABLE_HDF5)
   endif ()
 
   # Check if HDF5 version is 1.8 or greater
-  if (HDF5_VERSION VERSION_LESS "1.8.0")
```

Comment:
I haven't found any cmake documentation about HDF5_VERSION_STRING ... while HD5_VERSION should be properly defined.

Reply:
Correct, It should be HDF5_VERSION, which is set by find_package HDF5

Reply:
After updating to HDF5 version 1.10.9 the error I got doesn't occur, so yes the correct variable should be HDF5_VERSION. I'll push up the change.

Reply:
> What happens for the case CGNS_ENABLE_CGNSTOOLS=ON, CGNS_ENABLE_UTILS=OFF. Does it still build the command line tools?

The additional CMake commands should not impact the CGNS_ENABLE_CGNSTOOLS option. Yes CGNSTOOLS should be able to compile the tools. UTILS just allows for compiling the command line tools without the Tk/Tcl GUI.